requirements/base.txt

```diff
@@ -1,46 +1,46 @@
 # web application
 Django==3.2.11
-django-crispy-forms==1.12.0
-django-filter==2.4.0
-django-redis==5.0.0
-django-reversion==4.0.0
+django-crispy-forms==1.13.0
+django-filter==21.1
+django-redis==5.2.0
+django-reversion==4.0.1
 django-selectable==1.3.0
-django-taggit==1.5.1
+django-taggit==2.0.0
 django-treebeard==4.5.1
-django-webpack-loader==1.1.0
-djangorestframework==3.12.4
-  uritemplate==3.0.1  # schema dependencies
-  pyyaml==5.4.1       # schema dependencies
+django-webpack-loader==1.4.1
+djangorestframework==3.13.1
+  uritemplate==4.1.1  # schema dependencies
+  pyyaml==6.0       # schema dependencies
 gunicorn==20.1.0
 
 # tasks
-celery==5.1.2
+celery==5.2.3
   billiard==3.6.4.0
-  kombu==5.1.0
+  kombu==5.2.3
 
 # utilities
-jsonschema==4.1.2
-lxml==4.6.3
-psycopg2-binary==2.9.1
-redis==3.5.3
-requests==2.26.0
-pydantic==1.8.2
-myst-parser==0.15.1
+jsonschema==4.4.0
+lxml==4.7.1
+psycopg2-binary==2.9.3
+redis==4.1.1
+requests==2.27.1
+pydantic==1.9.0
+myst-parser==0.16.1
 
 # computational
-numpy==1.21.1
-pandas==1.3.0
-  openpyxl==3.0.7
-plotly==5.1.0
-pyarrow==4.0.1
-scipy==1.7.0
-nltk==3.6.2
-scikit-learn==0.24.2
+numpy==1.22.1
+pandas==1.3.5
+  openpyxl==3.0.9
+plotly==5.5.0
+pyarrow==6.0.1
+scipy==1.7.3
+nltk==3.6.7
+scikit-learn==1.0.2
```

Comment:
I think there were some pytest deprecation warnings with updating this package - could you please make sure those methods work and those deprecation warnings are removed?